src/libs/auth.ts

```diff
@@ -31,6 +32,18 @@ import {
 import * as Utils from 'src/libs/utils';
 import { v4 as uuid } from 'uuid';
 
+export interface OidcUser extends User {
+  profile: B2cIdTokenClaims;
+}
+
+export interface B2cIdTokenClaims extends IdTokenClaims {
```

Comment:
At some point (not necessarily in this PR), it would be nice to add documentation to this type explaining what each of these fields contains.

Reply:
That's a good idea. https://github.com/broadinstitute/terraform-ap-deployments/tree/master/azure/b2c/policies is how we are defining them, at least I can link this above.